jobs/ci-kubernetes-e2e-gce-1.6-master-cvm-kubectl-skew-serial.env

```diff
@@ -0,0 +1,9 @@
+E2E_OPT=--check_version_skew=false
+GINKGO_PARALLEL=n
+GINKGO_TEST_ARGS=--ginkgo.focus=Kubectl.*\[Serial\]
+JENKINS_PUBLISHED_SKEW_VERSION=ci/latest
+JENKINS_PUBLISHED_VERSION=ci/latest-1.6
+PROJECT=k8s-gce-cvm-1-6-mstr-ctl-skew-serial
```

Comment:
we have work underway to stop treating projects like pets, which will help here.